src/ReanimatedModule.ts

```diff
@@ -0,0 +1,64 @@
+import { NativeModules } from 'react-native';
+import { nativeShouldBeMock } from './reanimated2/PlatformChecker';
+
+const ReanimatedModuleMock = {
+  async disconnectNodeFromView(): Promise<void> {
+    // noop
+  },
+  async attachEvent(
+    _viewTag: number,
+    _eventName: string,
+    _nodeID: number
+  ): Promise<void> {
+    // noop
+  },
+  async detachEvent(
+    _viewTag: number,
+    _eventName: string,
+    _nodeID: number
+  ): Promise<void> {
+    // noop
+  },
+  async createNode(
+    _nodeID: number,
+    _config: Record<string, unknown>
+  ): Promise<void> {
+    // noop
+  },
+  async dropNode(_nodeID: number): Promise<void> {
+    // noop
+  },
```

Comment:
Why do we need this method?

Reply:
You're right, I'll keep only the ones that actually are in use